redi-connect-front/src/components/molecules/ReadMentoring.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Content, Tag } from 'react-bulma-components'
+import { RedProfile } from '../../types/RedProfile'
+import { connect } from 'react-redux'
+import { RootState } from '../../redux/types'
+import { Caption } from '../atoms'
+import { categoriesIdToLabelMap } from '../../config/config'
+
+interface ReadMentoringProps {
+  profile: RedProfile
+  caption?: boolean
+}
+
+interface TagsProps {
+  items: string[]
+}
+
+const Placeholder = () => <Content italic>Please pick up to three mentoring topics.</Content>
+
+const ProfileTags = ({ items }: TagsProps) => (
+  <Tag.Group>
+    {items.map(tagId =>
+      <Tag key={tagId} size="large" rounded>
+        {categoriesIdToLabelMap[tagId]}
+      </Tag>
+    )}
+  </Tag.Group>
+)
+
+const ReadMentoring = ({ profile, caption }: ReadMentoringProps) => {
```

Comment:
Let's also make this one more explicit. "Mentoring" by itself can mean a lot of things. "MentoringTopics" or "MentoringCategories" is more to the point.